src/cache-view.ts

```diff
@@ -161,46 +243,83 @@ export class ConfigurationWebview {
-            color: white;
+            background: var(--vscode-button-background);
+            color: var(--vscode-button-foreground);
             text-transform: uppercase;
             font-weight: bold;
-            border: 1px solid #ddd;
+            border: solid 1px var(--vscode-contrastBorder);
             transition: 100ms ease-in-out;
           }
-          .vscode-light button#save:hover {
-            cursor: pointer;
-            background: #ddd;
-          }
+          .vscode-light button#save:hover,
           .vscode-dark button#save:hover {
             cursor: pointer;
-            background: #333;
+            background-color: var(--vscode-button-hoverBackground);
+          }
+          .vscode-light button#save:focus,
+          .vscode-light button#save input:focus,
+          .vscode-light button#save select:focus,
+          .vscode-dark button#save:focus {
+            outline: 1px solid -webkit-focus-ring-color;
+            outline-offset: 2px;
+            border-color: var(--vscode-focusBorder);
+            border: var(--vscode-focusBorder);
+            contrastBorder: var(--vscode-focusBorder);
+            contrastActiveBorder: var(--vscode-focusBorder);
+          }
+          .vscode-light button#save:active,
+          .vscode-dark button#save:active {
+            outline: none;
+          }
+
+          checkbox:active,
+          checkbox {
+            color: var(--vscode-settings-checkboxForeground);
+            background: var(--vscode-settings-checkboxForeground);
+            border: var(--vscode-settings-checkboxBorder);
+          }
+
+          .vscode-light cmake-input-bool input:checked,
+          .vscode-dark cmake-input-bool {
+            color: var(--vscode-settings-checkboxForeground);
+            background: var(--vscode-settings-checkboxForeground);
+            border: var(--vscode-settings-checkboxBorder);
+          }
+
+          .cmake-input-bool input:checked,
+          .cmake-input-bool:active,
+          .cmake-input-bool {
+            color: var(--vscode-settings-checkboxForeground);
+            background: var(--vscode-settings-checkboxForeground);
+            border: var(--vscode-settings-checkboxBorder);
+          }
+          a:focus,
+          input:focus,
+          select:focus,
+          textarea:focus {
+              outline: 1px solid -webkit-focus-ring-color;
+              outline-offset: -1px;
+              border-color: var(--vscode-focusBorder);
+              contrastBorder: var(--vscode-focusBorder);
+              contrastActiveBorder: var(--vscode-focusBorder);
           }
         </style>
         <script>
           const vscode = acquireVsCodeApi();
           function toggleKey(id) {
             const label = document.getElementById('LABEL_' + id);
-            label.textContent = label.textContent == 'ON' ? 'OFF' : 'ON';
+            label.textContent = label.textContent == '${onButtonText}' ? '${offButtonText}' : '${onButtonText}';
```

Comment:
I think these are CMake enum values.  I don't believe they should be localized.

Reply:
Ups, yes